ForkMe-Mobile/app/src/main/java/dlei/forkme/gui/activities/github/MergeMeActivity.java

```diff
@@ -0,0 +1,212 @@
+                            }
+                        }
+                );
+            } else {
+                // Can give permissions in app.
+                mMainTextView.setText(R.string.give_perms_in_app_text);
+                mMainButton.setText(R.string.give_perms_in_app_button);
+                final Context context = this;
+                mMainButton.setOnClickListener(
+                        new View.OnClickListener() {
+                            @Override
+                            public void onClick(View v) {
+                                Intent i = new Intent(context, SettingsActivity.class);
+                                startActivity(i);
+                            }
+                        }
+                );
+            }
+        } else {
+            // Set up views.
+            mMainTextView.setVisibility(View.GONE);
+            mMainButton.setVisibility(View.GONE);
+            mProgressBarSpinner.setVisibility(View.VISIBLE);
+            mRecyclerViewDevContactInfo.setVisibility(View.VISIBLE);
+            // If developer list not populated, populate it.
+            if (!(mDeveloperContactInfo.size() > 0)) {
+                this.getDevelopers();
+            }
+        }
+    }
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        Log.d("MergeMeActivity: ", "creating");
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_merge_me);
+        super.inflateNavDrawer(savedInstanceState, MergeMeActivity.class.getSimpleName());
+        Log.d("MergeMeActivity: ", "created");
+        setTitle("MergeMe");
+        // Set up UI elements.
+        mRecyclerViewDevContactInfo = (RecyclerView) findViewById(R.id.mergeMeRecyclerView);
+        mMainButton = (AppCompatButton) findViewById(R.id.mergeMeMainButton);
+        mMainTextView = (AppCompatTextView) findViewById(R.id.mergeMeMainText);
+        mProgressBarSpinner = (ProgressBar) findViewById(R.id.progress_bar_spinner);
+
+        // Set up components of RecyclerView.
+        mAdapterDevContactInfo = new DeveloperContactRecyclerViewAdapter(mDeveloperContactInfo);
+        mLayoutManager = new LinearLayoutManager(this);
+
+        // Set up RecyclerView.
+        mRecyclerViewDevContactInfo.setLayoutManager(mLayoutManager);
+        mRecyclerViewDevContactInfo.setItemAnimator(new DefaultItemAnimator());
+        mRecyclerViewDevContactInfo.setAdapter(mAdapterDevContactInfo);
+
+        // Set up lines between items in list.
+        DividerItemDecoration dividerItemDecoration = new DividerItemDecoration(mRecyclerViewDevContactInfo.getContext(),
+                mLayoutManager.getOrientation());
+        mRecyclerViewDevContactInfo.addItemDecoration(dividerItemDecoration);
+
+    }
+
+    /**
+     * Get developers who have been aggregated together based on location.
+     */
+    public void getDevelopers() {
+        OkHttpClient.Builder okHttpBuilder = new OkHttpClient.Builder();
```

Comment:
Thanks, just a note to self. Can do headers like so
<img width="807" alt="screen shot 2017-06-12 at 11 26 45 pm" src="https://user-images.githubusercontent.com/11433468/27035997-c473cc38-4fc6-11e7-8853-af953b0a5942.png">